### PR TITLE
[PLAT-10602] Add `sendPageAttributes` configuration option

### PR DESCRIPTION
**Pull request by gingerbenw**
## Goal

Add a new configuration option to manually exclude `url`, `title` and `referrer` attributes from automatically instrumented spans

## Testing

Unit tests for validation functions

### CODE REVIEW

File: packages/platforms/browser/lib/send-page-attributes.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+export function getPermittedAttributes (sendPageAttributes: SendPageAttributes) {
+  return {
+    ...defaultSendPageAttributes,
+    ...sendPageAttributes
+  }
+}
+
+export function isSendPageAttributes (obj: unknown): obj is SendPageAttributes {
+  return isObject(obj) && Object.values(obj).every(value => typeof value === 'boolean') && Object.keys(obj).every(key => Object.keys(defaultSendPageAttributes).includes(key))
```
[the remark this answers is not in the document]
Comment (gingerbenw) [reply packages/platforms/browser/lib/send-page-attributes.ts:23]: yeah, that probably is a little overkill. any additional properties won't be used, so we can slacken this up

[the remark this answers is not in the document]
Comment (gingerbenw) [reply packages/platforms/browser/lib/send-page-attributes.ts:23]: "is an object, and every key is boolean?"

[the remark this answers is not in the document]
Comment (gingerbenw) [reply packages/platforms/browser/lib/send-page-attributes.ts:23]: alternatively, "is an object, and `.referrer` `.title` and `.url` are either `boolean` or `undefined`". ?

[the remark this answers is not in the document]
Comment (yousif-bugsnag) [reply packages/platforms/browser/lib/send-page-attributes.ts]: yeah, I think we should allow undefined keys

**Review by yousif-bugsnag — APPROVED** [review]
LGTM

**Pull request merged**